cmsl1t/analyzers/BaseAnalyzer.py

```diff
@@ -88,4 +122,14 @@ def get_histogram_filename(self):
         return os.path.join(self.output_folder, output_file)
 
     def add_plotter(self, plotter):
+        """
+        Register a plotter with this analyzer, and set up it's outputs
```

Comment:
The actual task of this method evolved and I didn't update the name.  Will switch to `register_plotter`